src/zentables/zentables.py

```diff
@@ -916,6 +916,10 @@ def _do_suppression(
     Returns:
         pd.DataFrame where entries are True if df_n needs to be suppress and False if not.
     """
+    # See if there are any NaNs, and raise error
+    if df_n.isnull().values.any():
```

Comment:
Is this the correct behavior, or should you just the initial mask to the null values? I can be convinced either way, honestly.... Either way, should add to the doc